.github/workflows/run-tests.yml

```
@@ -271,6 +271,33 @@ jobs:
           echo "$TEST_RESULTS_WALLET" | base64 -d > wallet.json
         working-directory: neofs-testcases
 
+# For test reports from master the expiration period is 1 month
+# For test reports from PR the expiration period is 2 weeks
+# For test reports from releases - no expiration period
+# For test reports from manual run the expiration period is 2 weeks
+# For other test reports the expiration period is 1 week
+      - name: Define expire
+        if: always()
+        env:
+          TEST_RESULTS_PASSWORD: ${{ secrets.TEST_RESULTS_PASSWORD }}
+          TEST_RESULTS_NEOFS_NETWORK_DOMAIN: ${{ vars.TEST_RESULTS_NEOFS_NETWORK_DOMAIN }}
+        run: |
+          CURRENT_EPOCH=$(NEOFS_CLI_PASSWORD=$TEST_RESULTS_PASSWORD neofs-cli netmap epoch --rpc-endpoint st1.$TEST_RESULTS_NEOFS_NETWORK_DOMAIN:8080 \
+          --wallet wallet.json)
+          if [[ "${{ github.event_name }}" == "push" ]]; then
+            EXPIRE=$((744 + CURRENT_EPOCH))
```

```suggestion
            EXPIRE=$((24*31 + CURRENT_EPOCH))
```
to make it less magic. also, i would provide a comment about epoch longevity, it is not that const in fact

---

> to make it less magic. also, i would provide a comment about epoch longevity, it is not that const in fact

Fixed